app/representers/space_representer.rb

```diff
@@ -27,5 +27,11 @@ class SpaceRepresenter < Roar::Decorator
     property :admin_id
     property :name
     property :picture
+
+    property :plan_details,
```

Comment:
We actually do load all the users organizations with they sign in. We need to to render the organizations dropdown in the header. It's in the userOrganizationsMemberships actions file. 
I think tying into that mechanism would be a better way of loading this information.